Makefile

```diff
@@ -167,16 +167,16 @@ debug-shared: shared
 static: lib/libsass.a
 shared: $(LIBRARIES)
 
-lib/libsass.a: $(COBJECTS) $(OBJECTS)
+lib:
 	$(MKDIR) lib
```

Comment:
Nope, we do not care if the directory already exists, but the make process aborted if it tried to "re-create" an already existing directory. And there is no direct option to surpress it (the -p option lead to create directories named "-p" on windows). For now I resolved to create targets for the directory in the makefile. AFAIK these targets will only be executed if the path/file does not exist. I know that this is ok for files, but I'm not sure if this is also true for directories by definition! At least this PR seems to work on linux, mingw (windows) and mac os x!

Just tried your suggestion with `@$(MKDIR) lib || true` and it seems to be supported by windows! Seems like another possible solution to this problem. But still not sure what the best and most portable solution is!